lib/browser/guest-window-manager.ts

```diff
@@ -73,10 +74,15 @@ export function openGuestWindow ({ event, embedder, guest, referrer, disposition
     }
   }
 
-  const window = new BrowserWindow({
-    webContents: guest,
-    ...browserWindowOptions
-  });
+  let window: BrowserWindow;
+  if (overrideWindowCreationFunction) {
+    window = overrideWindowCreationFunction(browserWindowOptions, guest);
```

Comment:
```suggestion
    window = overrideWindowCreationFunction({
      webContents: guest,
      ...browserWindowOptions
    });
```

Reply:
Should we leave `webContents: guest` line above the `...browserWindowOptions`? `webContents` value will get overwritten by the value that comes in the `browserWindowOptions` object if `webContents` property of it is set.